### PR TITLE
[Bug Fix]LPS-129966 

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/model/listener/PortalInstanceLifecycleListenerImpl.java [hunk 1]
```diff
@@ -107,14 +107,19 @@ public void portalInstanceRegistered(Company company) throws Exception {
...
+
+					_dlAppLocalService.addFileEntry(
+						PropsKeys.IMAGE_DEFAULT_COMPANY_LOGO, user.getUserId(),
+						repository.getRepositoryId(),
+						DLFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+						"company_logo.png", mimeType, image.getTextObj(), null,
+						null, serviceContext);
+				}
+				finally {
+					FileUtil.delete(file);
```
Comment (tinatian) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/model/listener/PortalInstanceLifecycleListenerImpl.java:121]: FYI @alecsloan

**Comment by vicnate5** [comment]
ci:test:bundle

**Comment by vicnate5** [comment]
@yuhai @dantewang This passes manual testing on Linux, but fails testing on Windows.

Steps to Reproduce on a Windows machine
1. Download [bundle from CI ](https://files.liferay.com/private/bundles/test-1-4/jobs/test-portal-acceptance-pullrequest(master)/builds/8518/liferay-portal-bundle-tomcat.tar.gz)
2. Extract and remove portal-ext from `tomcat-9.0.43\webapps\ROOT\WEB-INF\classes`
3. Start portal and wait for full startup
4. Shutdown portal
5. Check contents of `tomcat-9.0.43\temp`


Result:
```
./20210908202205053ROOUVZMT
./20210908202205053ROOUVZMT/accounts
./20210908202205053ROOUVZMT/accounts/accounts_run.bat
./20210908202208113RVTYIJWY
./20210908202208113RVTYIJWY/orders
./20210908202208113RVTYIJWY/orders/orders_run.bat
./20210908202208954WQRAUBAW
./20210908202208954WQRAUBAW/price_list
./20210908202208954WQRAUBAW/price_list/price_list_run.bat
./20210908202209784SDSKQZBR
./20210908202209784SDSKQZBR/products
./20210908202209784SDSKQZBR/products/products_run.bat
```

**Comment by dantewang** [comment]
See https://github.com/tinatian/liferay-portal/pull/2380

ci:close

**Comment by dantewang** [comment]
ci:close

**Pull request closed without merging**